cfme/ansible_tower/jobs.py

```diff
@@ -26,6 +26,13 @@ def in_jobs(self):
         return (self.logged_in_as_current_user and
                 self.navigation.currently_selected == ['Automation', 'Ansible Tower', 'Jobs'])
 
+    @View.nested
+    class my_filters(Accordion):  # noqa
```

Comment:
How about defining this standard view in `widgetastic_manageiq` and using `View.nested(FilterAccordion)` in all of the places where its defined in this PR?

Reply:
I can work on refactoring, but merging this PR would save so much time on testing so I prefer to make it merged and then I can play with it and refactor these accordions. Thanks